funnel/templates/503.html.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "layout.html.jinja2" %}
```

Comment:
This template needs to be a static file in https://github.com/hasgeek/serverdown as this code will never be called. 503 is raised by Nginx.

However, in future Funnel may connect to an upstream server and may want to raise a 503 if it is unavailable, so we should keep the template here as well.

Reply:
https://github.com/hasgeek/serverdown/pull/1